### PR TITLE
Migrating from Bors to GitHub Merge queue + Buildkite

### PR DESCRIPTION
**Pull request by sriharshakandala**
## Purpose 
Migrating from Bors to GitHub Merge queue + Buildkite

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by szy21** [comment]
I think we can just use the option without merge queue, as we don't have many commits. What do you think?

**Comment by sriharshakandala** [comment]
> I think we can just use the option without merge queue, as we don't have many commits. What do you think?

Is there any disadvantage to using merge queue on repos with fewer commits? Maybe, it will help occasionally when there is a larger commit traffic!

**Comment by szy21** [comment]
> > I think we can just use the option without merge queue, as we don't have many commits. What do you think?
> 
> Is there any disadvantage to using merge queue on repos with fewer commits? Maybe, it will help occasionally when there is a larger commit traffic!

It just doubles the time to merge PRs, but the test for this repo is fast enough so I think it's ok too.

**Comment by szy21** [comment]
It seems we don't need to run build on main: https://github.com/CliMA/ClimaAtmos.jl/pull/2338#issuecomment-1797848260

File: .github/workflows/OS-UnitTests.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ on:
     branches:
       - staging
       - trying
+    tags: '*'
```
Comment (szy21) [inline .github/workflows/OS-UnitTests.yml:9]: remove the branches above?

**Comment by sriharshakandala** [comment]
> > > I think we can just use the option without merge queue, as we don't have many commits. What do you think?
> > 
> > 
> > Is there any disadvantage to using merge queue on repos with fewer commits? Maybe, it will help occasionally when there is a larger commit traffic!
> 
> It just doubles the time to merge PRs, but the test for this repo is fast enough so I think it's ok too.

Ok, I removed the merge queue option.

**Review by szy21 — APPROVED** [review]

**Pull request merged**